src/style.css

```diff
@@ -0,0 +1,29 @@
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
```

Comment:
flex-direction is redundant
all blocks are started from the new line by default
same with justify-content: delete it